tests/counterpoll_test.py

```diff
@@ -17,16 +17,16 @@
 
 import counterpoll.main as counterpoll
 
-expected_counterpoll_show = """Type                    Interval (in ms)  Status
+expected_counterpoll_show = """Type                  Interval (in ms)    Status
 --------------------  ------------------  --------
-QUEUE_STAT                         10000  enable
-PORT_STAT                           1000  enable
-PORT_BUFFER_DROP                   60000  enable
-QUEUE_WATERMARK_STAT               10000  enable
-PG_WATERMARK_STAT                  10000  enable
-PG_DROP_STAT                       10000  enable
-ACL                                10000  enable
-FLOW_CNT_TRAP_STAT                 10000  enable
+QUEUE_STAT            10000               enable
```

Comment:
This format is different with the output of counterpoll show on tb as bellowing, wondering if this test case can pass?

admin@str2-7260cx3-acs-12:~$ counterpoll show
Type                        Interval (in ms)    Status
--------------------------  ------------------  --------
QUEUE_STAT                  default (10000)     enable
PORT_STAT                   default (1000)      enable
PORT_BUFFER_DROP            default (60000)     enable
RIF_STAT                    default (1000)      enable
QUEUE_WATERMARK_STAT        default (10000)     enable
PG_WATERMARK_STAT           default (10000)     enable
PG_DROP_STAT                default (10000)     enable
BUFFER_POOL_WATERMARK_STAT  default (10000)     enable
admin@str2-7260cx3-acs-12:~$

Reply:
Yes. that's because in the testbed all our default values but here some are configured and some are default. python wheel was built successfully in the pr run